public/js/saveApplicant.js

```diff
@@ -73,6 +79,91 @@ let errorMessage = (validationType) => {
     return htmlString;
 };
 
+let getInProgressFormData = () => {
+    const formData = document.querySelectorAll(".submitApplicant");
+    let stagedData = {}
+    stagedData['cohort'] = [];
+    formData.forEach(formItem => {
+        if (formItem.id !== 'termsAndConditionsCheck') {
+            if (formItem.type === 'checkbox') {
+                if (formItem.name === 'cohort') {
+                    if (formItem.checked) {
+                        stagedData[formItem.name].push(formItem.value);
+                    }
+                } else {
+                    stagedData[formItem.name] = formItem.checked;
+                }
+            } else {
+                stagedData[formItem.name] = formItem.value;
+            }
+        }
+    })
+    Object.keys(stagedData).forEach(stagedItemKey => {
+        console.log(stagedItemKey, stagedData[stagedItemKey])
+        window.localStorage.setItem(stagedItemKey, stagedData[stagedItemKey])
```

Comment:
No need to do window.localstorage, the window object is the global object which means you can just do `localstorage.setItem()`